backendStub/service/FrikkService.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+
+/**
+ * Return data of a specific menu
+ * Returns all the data about the menu with given menuID, even the menu items inside it. The photos of the menu items are saved in the Amazon s3 storage, so the links to the cloud storage are also returned. The frontend will directly download them from the cloud storage, they won't be sent by the backend with this endpoint.
+ *
+ * menuID Integer ID of the menu to be returned
+ * returns MenuRetrievedToCustomer
+ **/
+exports.getMenuCustomer = function(menuID) {
+  return new Promise(function(resolve, reject) {
+    var examples = {};
+    examples['application/json'] = {
+  "restaurant" : {
+    "restaurantName" : "Emilio's Pizza",
+    "address" : "André route 45",
+    "city" : "Rome",
+    "country" : "Italy",
+    "latitude" : 48.4281350802915,
+    "longitude" : -110.0792542197085
+  },
+  "menuName" : "Emilio's menu of the day",
+  "description" : "Our special menu of today",
+  "tags" : [ {
+    "name" : "Italian",
+    "color" : "#FFBC8C"
+  }, {
+    "name" : "Mediterranean",
+    "color" : "#FFBC8C"
+  } ],
+  "menuRating" : 5,
+  "menuItems" : [ {
+    "name" : "Spaghetti alla carbonara",
+    "description" : "Fantastic italian dish made of spaghetti, pig cheek, eggs, black pepper, pecorino romano",
+    "type" : "dish",
+    "priceEuros" : 10,
+    "tags" : [ {
+      "name" : "Mediterranean",
+      "color" : "#FFBC8C"
+    }, {
+      "name" : "Pasta",
+      "color" : "#99C99B"
+    }, {
+      "name" : "Italian",
+      "color" : "#FFBC8C"
+    } ],
```

Comment:
Can you add this tag as well? We would like to test the maximum number of tags which is 4. 
`,
      {
        "name" : "Dinner",
        "color" : "#99C99B"
      }`